pkg/apis/apiserver/organization_REST.go

```diff
@@ -361,12 +406,7 @@ func (o *OrganizationREST) isVisible(ctx context.Context, organization *Organiza
 		return true, nil
 	}
 
-	for _, sub := range append(
-		organization.Status.Members,
-		// This is important for seeing organizations you own before controller syncs status
-		// otherwise a watch misses create event
```

Comment:
Why do you remove this comment in this PR? I still think this is important, and if you remove it, may lead to some confusion in the future.

Reply:
by accident